pkg/configurator/methods_test.go

```diff
@@ -373,6 +381,9 @@ var _ = Describe("Test Envoy configuration creation", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			Expect(cfg.IsTracingEnabled()).To(BeFalse())
+			Expect(cfg.GetTracingHost()).To(Equal(constants.DefaultTracingHost + ".-test-osm-namespace-.svc.cluster.local"))
```

Comment:
Just curious - how does `-test-osm-namespace-` override the templated default? Is that set as the ns for this test/will it not default to "namespaceName" in line 36?

Reply:
The client used for this test is defined here:
https://github.com/openservicemesh/osm/blob/250c9c22dd5cec75d9a0afb72a9be5896b6dbd24/pkg/configurator/methods_test.go#L321

which uses the namespace defined here:
https://github.com/openservicemesh/osm/blob/250c9c22dd5cec75d9a0afb72a9be5896b6dbd24/pkg/configurator/client_test.go#L20

Reply:
gotcha, thanks for explaining. GH search gave me a million results that didn't match this string